octobot_evaluators/evaluator/social_evaluator.py

```diff
@@ -27,10 +27,7 @@ class SocialEvaluator(AbstractEvaluator):
     def __init__(self):
         super().__init__()
         self.load_config()
-
-    @classmethod
-    def get_is_symbol_widlcard(cls) -> bool:
-        return False
+        self.exchange_id = None
```

Comment:
I don't think a `social_evaluator` is linked to only one exchange

Reply:
For now we create them for each exchange since ```create_all_type_evaluators``` is called for each exchange.
Based on the exchange name it has, it pushes in the matrix accordingly (under this exchange). That was the reason of the octobot-trading method that was giving a random exchange based on matrix ID, then I realized an exchange name in available. 
We should update the evaluator creation process not to create social evaluators each time, but we also need a way to check if they are already created. 
I suggest we do it in another PR.

Regarding this ```self.exchange_id```, we could keep something similar as a buffer for social evaluators not to have to iterate over all exchanges (in order to find an exchange with the right matrix ID to ask for the current time). This buffer would basically be empty at first but after the 1st eval it would contain an exchange name and id allowing to find it faster next time we need an exchange time (for the next eval publication basically)

Reply:
Ok I didn't remember that social_evaluators are created for each exchanges. 
For now I think we should keep this implementation but we have to refactor this later with an exchange independant social_evaluator.